packages/core/src/odata-v2/request/odata-function-import-request-config.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  ODataUri,
+  FunctionImportParameter,
+  FunctionImportParameters,
+  RequestMethodType,
+  ODataFunctionImportRequestConfig
+} from '../../odata-common';
+
+export class ODataFunctionImportRequestConfigV2<
+  ParametersT
+> extends ODataFunctionImportRequestConfig<ParametersT> {
+  /**
+   * Creates an instance of ODataFunctionImportRequestConfig.
+   *
+   * @param method - HTTP method for the request
+   * @param defaultServicePath - Default path of the service
+   * @param functionImportName - The name of the function import.
+   * @param parameters - Object containing the parameters with a value and additional meta information
+   * @param oDataUri - ???
```

Comment:
Please fix it by e.g. searching for the doc of `oDataUri` in other files

Reply:
Oops, forgot about that.